templates/sdk-github-issue-template.yaml

```diff
@@ -0,0 +1,48 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug", "triage"]
+body:
+  - type: input
+    id: library-version
+    attributes:
+      label: Library version
+      description: What version of the library are you using?
+      placeholder: ex. v1.2.21
+    validations:
+      required: true
+  # Can remove this field for most SDKs, but useful for JS
+  - type: input
+    id: library-package
+    attributes:
+      label: Library package
+      description: What package are you using within the library?
+      placeholder: e.g. Callbacks, promises
+    validations:
+      required: true
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: What went wrong? Tell us what you see!
+    validations:
+      required: true
+  - type: textarea
+    id: expected-behaviour
+    attributes:
+      label: What did you expect to happen?
+      description: If things were working as you'd expect, what would have you seen instead?
+    validations:
+      required: true
+  - type: textarea
```

Comment:
My personal preference would be to  merge these two fields into an "additional information" field which suggests sharing a minimal repro or debug logs